### DISCUSSION TITLE
Image | Libre Computer Board AML-S905X-CC (Le Potato)

### DISCUSSION THREAD

**Discussion by hoser24** [MichaIng/DietPi#5711] [Image Request]
I would like to say thank you for hard work that has been done to make dietpi possible and also make an image request for the libre computing board le potato it looks like it would be a server which is actually what i would like to use for. thanks have a nice day 

specs for libre's le potato https://libre.computer/products/s905x/

**Comment by nonnorm** [comment]
You should be able to use the DietPi installer script with the Armbian image at https://www.armbian.com/lepotato/ (make sure to use Debian Bullseye instead of Ubuntu Jammy):
`bash -c "$(curl -sSfL https://raw.githubusercontent.com/MichaIng/DietPi/master/.build/images/dietpi-installer)"`

**Comment by hoser24** [comment]
thanks for the suggestion ravenclaw900 but the bullseye image turned out to be android box unfortunately

**Comment by coolfdu1** [comment]
Please see this link for debian bullseye:
https://sd-card-images.johang.se/boards/libretech_s905x_cc.html

**Reply by coolfdu1** [reply]
I have prepared an image with Armbian Bullseye and the script above.
https://www.coolfdu1.com/static/DietPi_LePotatov1-ARMv8-Bullseye.7z

**Reply by coolfdu1** [reply]
Updated image for Armbian Bookworm.
[https://www.coolfdu1.com/static/DietPi_LePotatoV1-ARMv8-Bookworm.img.xz](https://www.coolfdu1.com/static/DietPi_LePotatoV1-ARMv8-Bookworm.img.xz)

SHA256:
[https://www.coolfdu1.com/static/DietPi_LePotatoV1-ARMv8-Bookworm.img.xz.sha256](https://www.coolfdu1.com/static/DietPi_LePotatoV1-ARMv8-Bookworm.img.xz.sha256)

**Reply by drake90001** [reply]
> Updated image for Armbian Bookworm. https://www.coolfdu1.com/static/DietPi_LePotatoV1-ARMv8-Bookworm.img.xz
> 
> SHA256: https://www.coolfdu1.com/static/DietPi_LePotatoV1-ARMv8-Bookworm.img.xz.sha256

outdated link?

**Comment by jtmoon79** [comment]
Relates to #6932 #6707
